angular-workspace/package.json

```diff
@@ -60,6 +60,6 @@
     "ng-packagr": "^15.2.2",
     "playwright": "1.40.0",
     "rollup": "^4.12.0",
-    "typescript": "~4.8.2"
+    "typescript": "~4.9.5"
```

Comment:
[Angular's docs](https://angular.io/guide/versions#actively-supported-versions) show that this version of TypeScript is compatible with Angular 15.2 👍. Could you mention that in the PR description?